### PR TITLE
Spec the beacon fetch integration

### PR DESCRIPTION
**Pull request by domfarolino**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/fenced-frame/pull/106.html" title="Last updated on Jun 24, 2023, 7:11 PM UTC (176cbb5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/fenced-frame/106/376d4d0...176cbb5.html" title="Last updated on Jun 24, 2023, 7:11 PM UTC (176cbb5)">Diff</a>

### CODE REVIEW

File: spec.bs
Comment (domfarolino) [inline spec.bs:776]: This is the only thing that is blocking this PR I think. From [this](https://wicg.github.io/fenced-frame/#ref-for-send-a-beacon%E2%91%A0) entrypoint I can get an origin pretty easily and plumb it through. But I don't know enough about what's going on with https://wicg.github.io/fenced-frame/#finalize-a-reporting-destination or how to get an origin from there. Thoughts? Do we need to store it somewhere in the destination map or something?

File: spec.bs
Comment (gtanzer) [inline spec.bs:749]: Is there supposed to be a noun after "following"?

**Review by gtanzer — APPROVED** [review]
LGTM, though I don't really know the details of how the request should be configured.

File: spec.bs
Comment (gtanzer) [reply spec.bs:776]: Ah, thought this would be in a followup.

I assume it needs to be stored in the pending event, like in the impl https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:content/browser/fenced_frame/fenced_frame_reporter.cc;l=288;drc=fcf80d7b62173cb0f418679551f0dd8909cb79ab;bpv=1;bpt=1

Comment (gtanzer) [reply spec.bs:776]: https://wicg.github.io/fenced-frame/#fencedframetype-pending-event

so just another field here, and stored wherever the rest is currently stored

File: spec.bs
Comment (domfarolino) [reply spec.bs:749]: Meh, idk. Requests aren't formal structs, so I don't want to say "items". I guess I could put "properties".

File: spec.bs
Comment (domfarolino) [reply spec.bs:776]: OK we can probably tackle that separately then. I'll file a bug for this outside of this PR and we can unblock this one.

**Pull request merged**
